ftplugin/forth.vim

```diff
@@ -0,0 +1,29 @@
+" Vim filetype plugin file
+" Language:	Forth
+" Maintainer:	Johan Kotlinski <kotlinski@gmail.com>
+" Last Change:	2023 Apr 12
+
+if exists("b:did_ftplugin")
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:cpo_save = &cpo
+set cpo&vim
+
+setlocal commentstring=\\\ %s
+setlocal comments=b:\\,s:.(,mb:\ ,e:)
+setlocal formatoptions-=t formatoptions+=croql
+
```

Comment:
i think it would be nice to remove this empty line, don't you agree, to have four setlocals in a row?

Reply:
Done.

It probably made more sense in my local ftplugin.  I pulled out some things like b:match_words, 'define' and 'include' which I'll include in another PR when I clean them up.